backend/src/main/java/de/amos/apachepulsarui/service/ConsumerService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: 2023 Niklas Teschner <niklas.teschner@web.de>
+ */
+
+package de.amos.apachepulsarui.service;
+
+import org.apache.pulsar.common.policies.data.ConsumerStats;
+import org.apache.pulsar.common.policies.data.TopicStats;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ConsumerService {
```

Comment:
I like that you created a new service class, makes everything clearer. I was wondering if we should do the same for the producer stuff?

Reply:
I should be you are right. I was a bit busy so ill do it in the next sprint :)